src/Lecture1.hs

```diff
@@ -119,4 +137,21 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+
+-- First take, where I traverse a list ~ three(?) times
+lowerAndGreater' :: (Show a, Ord a) => a -> [a] -> [Char]
+lowerAndGreater' n list =
+    show n ++ " is greater than " ++ show lower ++ " elements and lower than " ++ show greater ++ " elements"
+    where lower = length $ [x | x <- list, x < n ]
+          greater = length $ [x | x <- list, x > n ]
+
+-- Second take
```

Comment:
Performance-wise, this second implementation is better because it traverses the list only once. And it's usually better to implement functions that work with lists in a single-traversal fashion. This will allow to compose functions nicer.